source/_integrations/sensibo.markdown

```
@@ -45,127 +45,193 @@ then in the Sensibo app log you will be able to distinguish between actions
+| Main sensor                                | Binary sensor        | Is the connected motion sensor the main sensor for the Air device.                |
+| Room occupied                              | Binary sensor        | Is there presence in the room of the Air device.                                  |
+| Temperature                                | Sensor               | Temperature reading.                                                              |
+| Humidity                                   | Sensor               | Humidity reading.                                                                 |
+| Battery voltage                            | Sensor               | Voltage from battery.                                                             |
+| RSSI                                       | Sensor               | RSSI reading from connectivity.                                                   |
 
 ## Custom actions
 
 ### Get device mode capabilities
 
 As the below custom actions [Full state](#full-state) and [Climate react](#climate-react) both require their inputs to be exactly what the API requires, this custom action will provide the capabilities for the device for a certain HVAC mode to help the users on using those actions properly.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)